<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip"            default="192.168.1.100"/>
  <arg name="load_gripper"        default="true" />

  <include file="$(find franka_control)/launch/franka_control.launch">
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="false" />
  </include>
  <!-- <include file="$(find panda_moveit_config)/launch/panda_moveit.launch">
    <arg name="load_gripper" value="false" />
  </include> -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="position_joint_trajectory_controller"/>
  <include file="$(find panda_moveit_config)/launch/move_group.launch">
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>
  <!-- <node name="move_to_start" pkg="franka_example_controllers" type="move_to_start.py" output="screen" required="true" /> -->
</launch>
